hardhat.config.ts

```diff
@@ -7,15 +7,17 @@ import dotenv from 'dotenv'
 
 dotenv.config()
 
-let commonConfig = { gas: 5_000_000, gasPrice: 250_000_000_000 }
+let commonConfig = {}
```

Comment:
Is there a reason to move definition inside the conditional branch?

Reply:
there were error from my environment so I had to change it..
<img width="754" alt="image" src="https://github.com/Bisonai/data-feed-consumer/assets/148735107/bd571578-3ca6-4f89-b033-7731780e19ca">

Reply:
It complains about `accounts`, not about `gas` and `gasPrice`. Did you try to define accounts as below?

```typescript
let commonConfig = { gas: 5_000_000, gasPrice: 250_000_000_000, accounts: [] }
```

Then we could just update the `accounts`

```typescript
commonConfig.accounts = [process.env.PRIV_KEY]
```

Reply:
don't know what's different from my environment but it still leads to error
<img width="565" alt="Screenshot 2024-01-10 at 2 59 50 PM" src="https://github.com/Bisonai/data-feed-consumer/assets/148735107/cb9008dc-e968-4342-82b5-b530cd429e3e">
<img width="433" alt="image" src="https://github.com/Bisonai/data-feed-consumer/assets/148735107/6f259245-9dcc-4fa7-b168-4afb1436adc1">

Reply:
![Uploading image.png…]()

Reply:
You don't have to waste time on it. We can fix it later ;)